editor/import/resource_importer_scene.cpp

```diff
@@ -372,6 +372,194 @@ static void _pre_gen_shape_list(Ref<ImporterMesh> &mesh, Vector<Ref<Shape3D>> &r
 	}
 }
 
+struct ScalableNodeCollection {
+	List<Node3D *> node_3ds;
+	List<Ref<ImporterMesh>> importer_meshes;
+	List<Ref<Skin>> skins;
+	List<Ref<Animation>> animations;
+};
+
+void _rescale_importer_mesh(float p_scale, Ref<ImporterMesh> p_mesh, bool is_shadow = false) {
+	// MESH and SKIN data divide, to compensate for object position multiplying.
+
+	int surf_count = p_mesh->get_surface_count();
+	struct LocalSurfData {
+		Mesh::PrimitiveType prim;
+		Array arr;
```

Comment:
The naming `arr` comes from:
```
Array arr = p_mesh->get_surface_arrays(surf_idx);
```
This is currently the conventional name in Godot to use for Array objects holding the surface vertex and index buffers. Other naming suggestions may be welcome but this would affect a lot of import or mesh code.